depthai_sdk/src/depthai_sdk/oak_camera.py

```diff
@@ -164,6 +170,7 @@ def create_nn(self,
     def create_stereo(self,
                       resolution: Union[None, str, dai.MonoCameraProperties.SensorResolution] = None,
                       fps: Optional[float] = None,
+                      clickable: bool = True,
```

Comment:
I wouldn't expose this minor setting when creating Stereo - should be configurable from Visualizer

Reply:
Moved it to
```python 
visualizer.output(clickable=True)
```